### DISCUSSION TITLE
Unable to run Grid-Stat on EC2 instance for tutorial

### DISCUSSION THREAD

**Discussion by jwraby** [dtcenter/METplus#2160] [Training] [answered]
Following the tutorial at:
https://dtcenter.org/metplus-practical-session-guide-version-5-0/session-1-metplus-setupgrid-grid/met-tool-grid-stat/run

The first run in my EC2 instance using the provided command had the following error:
grd_file_type() -> file does not exist "/d1/METplus-5.0.0_Tutorial/output/met_output/pcp_combine/sample_fcst_12L_2005080712V_12A.nc"

I checked the path the sample fcst file and the directory named pcp_combine does not exist. The only directory at /d1/METplus-5.0.0_Tutorial/output/met_output/ is grid_stat.

Pls advise on how to proceed from here. 

Thanks.

**Comment by jprestop** [comment, marked as answer]
Hi @jwraby.  @georgemccabe has kindly pointed out to me that the input files for the Grid-Stat example are generated by the PCP-Combine example that is run earlier in the tutorial:
[SESSION 1: METPLUS SETUP/GRID-TO-GRID > MET TOOL: PCP-COMBINE](https://dtcenter.org/metplus-practical-session-guide-version-5-0/session-1-metplus-setupgrid-grid/met-tool-pcp-combine)
So, it looks like you will need to run through that section first.

I hope that helps!

**Reply by jwraby** [reply]
Thanks Julie and George. That explains it.
